### PR TITLE
Updated code snippet according to refactoring

### PR DESCRIPTION
**Pull request by TiagoBrenck**
## Purpose
Updating the first chapter code snippets to reflect the changes done on Microsoft.Identity.Web refactoring. This is an immediate action, but a full review must be done in all READMEs since we dont use AzureAD.UI methods anymore.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: 1-WebApp-OIDC/1-1-MyOrg/README.md [hunk 1]
```diff
@@ -164,7 +164,8 @@ cd "1-WebApp-OIDC\1-1-MyOrg"
      by this line:
 
      ```CSharp
-      services.AddSignIn(Configuration);
+      services.AddAuthentication(OpenIdConnectDefaults.AuthenticationScheme)
+              .AddSignIn("AzureAd", Configuration, options => Configuration.Bind("AzureAd", options));
```
Comment (jmprieur) [inline 1-WebApp-OIDC/1-1-MyOrg/README.md]: @TiagoBrenck:
Why do we need both the section and the delegate.
This should be one or the other?

Comment (jmprieur) [reply 1-WebApp-OIDC/1-1-MyOrg/README.md]: cc: @jennyf19 @pmaytak

Comment (TiagoBrenck) [reply 1-WebApp-OIDC/1-1-MyOrg/README.md]: The section is to configure `MicrosoftIdentityOptions.cs`. 

The delegate is the `Action<OpenIdConnectOptions>` where developers would be working with. With this, we are fully leveraging OpenIdConnect.

**Comment by TiagoBrenck** [comment]
Ok Jean-Marc. I will keep this discussion opened in: https://github.com/AzureAD/microsoft-identity-web/issues/37

**Pull request merged**